document_generator/generator_arguments.py

```diff
@@ -35,7 +35,8 @@ def get_mysql_conn():
         host=mysql_host,
         user=mysql_user,
         password=mysql_pass,
-        database=os.getenv("MYSQL_DATABASE", "ht")
+        database=os.getenv("MYSQL_DATABASE", "ht"),
+        pool_size=5
```

Comment:
We might be able to use a smaller pool size (e.g. just 1 might be fine) since we aren't using the connections in other threads, we just want to benefit from the automatic reconnection